pom.xml

```diff
@@ -197,6 +197,7 @@
   <modules>
     <module>library</module>
     <module>contrib</module>
+    <module>stream</module>
```

Comment:
should move this to the all-modules section because the stream module depends on kafka, which is in the all-modules section and is not built by default.